package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Setup the Pulumi CLI in a GitHub Actions build environment",
   "main": "dist/index.js",
   "scripts": {
-    "package": "ncc build src/main.ts -o dist --source-map --license LICENSE",
+    "package": "ncc build -o dist/setup-pulumi src/setup-pulumi.ts --source-map --license LICENSE && ncc build -o dist/cache-save src/cache-save.ts --source-map --license LICENSE",
```

Comment:
Maybe these two commands could be on separate scripts?

eg. "package:setup-pulumi", and they could be combined in `package`? My motivation here is to make it clearer/more readable. Any other suggestions would be great too!